docs/migration.md

```diff
@@ -506,6 +506,15 @@ except ValidationError as e:
     """
 ```
 
+#### Patterns / regex on strings
+
+Pydantic V1 used Python's regex library. Pydantic V2 uses the Rust [regex crate].
```

Comment:
Hmm... Should we add between parenthesis what "crate" means?

Reply:
I don't think it matters, if someone cares they can click the link.